krun/platform.py

```diff
@@ -815,6 +845,93 @@ def get_reboot_cmd(self):
         cmd.append(self.REBOOT)
         return cmd
 
+    def pin_process_args(self):
+        """Pin to a set of adaptive tick CPUs.
+        We are working the assumption that the kernel is NO_HZ_FULL_ALL meaning
+        that all but the first CPU are in adaptive tick mode."""
+
+        if self.num_cpus == 1:
+            fatal("not enough CPUs to pin")
+
+        cpus = ",".join([str(x) for x in xrange(1, self.num_cpus)])
+        return ["taskset", "-c", cpus]
+
+    def _check_util_linux_installed(self):
+        from distutils.spawn import find_executable
+        if not find_executable("taskset"):
+            fatal("util-linix is not installed "
+                  "(needed for pinning and scheduler tweaking).")
+
+    def sanity_checks(self):
+        UnixLikePlatform.sanity_checks(self)
+        self._check_util_linux_installed()
+        self._check_isolcpus()
+        self._sanity_check_cpu_affinity()
+        self._sanity_check_scheduler()
```

Comment:
That sounds like something we should document, but I'm not sure where. If we can't think of a good place I will raise an issue.